linux/setup/test_setup.yml

```diff
@@ -53,3 +53,20 @@
 # Create a base snapshot if it does not exist
 - include_tasks: create_base_snapshot.yml
   when: not (base_snapshot_exists | bool)
+
+# Hit issue in Ubuntu 22.04 casually: Temporary failure in name resolution. Restarting servcie can fix it
+- block:
+    - name: Get DNS servers
```

Comment:
Please quoted task name and add a name for the block.

Reply:
Done. thanks